spyder/api/shellconnect/main_widget.py

```diff
@@ -150,8 +276,22 @@ def remove_shellwidget(self, shellwidget):
 
             self.update_actions()
 
-    def set_shellwidget(self, shellwidget):
-        """Set widget associated with shellwidget as the current widget."""
+    def set_shellwidget(
+        self,
+        shellwidget: spyder.plugins.ipythonconsole.widgets.shell.ShellWidget,
+    ) -> None:
+        """
+        Set the given shell widget as the one associated to the current widget.
+
+        Parameters
+        ----------
+        shellwidget : spyder.plugins.ipythonconsole.widgets.shell.ShellWidget
+            The shell widget to associate with the current widget.
```

Comment:
Per the above, describe this correctly:

```suggestion
            The shell widget that corresponds to the stacked widget to set
            as currently active.
```